app/views/claims/_school_search.html.erb

```diff
@@ -26,7 +26,7 @@
       <div id="school-search-container">
 
         <%= errors_tag current_claim, :school_search %>
-        <%= text_field_tag :school_search, params[:school_search], id: :tslr_claim_school_search, class: css_classes_for_input(current_claim, :school_search), value: school_search_value %>
+        <%= text_field_tag :school_search, params[:school_search], id: :school_search, class: css_classes_for_input(current_claim, :school_search), value: school_search_value %>
```

Comment:
Could we use `errored_field_id_overrides on`render("shared/error_summary")` to set the error message anchor to `school_serach` to make the error message link to the input field?

Reply:
Good spot!